dashboard/app/views/admin_reports/admin_stats.html.haml

```diff
@@ -65,27 +55,28 @@
         %li
           30-Day Male Students:
           = @stats['actives']['thirty_day_male_students']
-    - ["historical", "current_weekly", "current_monthly"].each do |type|
+    - {'historical': 'accounts at least 37 days old and logged in at least once after 30 days',
+       'current_weekly': 'accounts 37 to 44 days old and logged in at least once after 30 days',
+       'current_monthly': 'accounts 60 to 90 days old and logged in at least once after 30 days'}.each do |type, description|
```

Comment:
on the one hand, this feels particularly verbose to declare here in the template. On the other hand, it's a display property, so it doesn't feel right to declare it in the view, either.

It might be cleaner to move this section out into a partial